services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -86,6 +86,10 @@ func (s PSMDBClusterService) ListPSMDBClusters(ctx context.Context, req *dbaasv1
 			State: psmdbStates()[c.State],
 		}
 
+		if c.Params.Paused {
```

Comment:
Until it's really paused I think status should be CHANGING

Reply:
so I think we should check if paused and cluster is ready.

Reply:
Fixed.